unidist/core/backends/mpi/core/executor.py

```diff
@@ -225,6 +230,51 @@ def is_already_sent(self, data_id, rank):
             rank in self._sent_data_map[data_id]
         )
 
+    def cache_serialization_info(self, data_id, data):
+        """
+        Save communication event for this `data_id` and `data`.
+
+        Parameters
+        ----------
+        data_id : unidist.core.backends.mpi.core.common.MasterDataID
+            An ``ID`` to data.
```

Comment:
```suggestion
            An ID to data.
```
We haven't `ID` entity